Gruntfile.js

```diff
@@ -216,17 +233,25 @@ module.exports = function(grunt) {
       grunt.fail.fatal('You must specify a release type. i.e. grunt release:prerelease');
     }
 
-    grunt.task.run([
-      'prompt:gh-release',
-      'build',
-      'test:unit',
-      'copy:dist',
+    var dryRun = grunt.option('dry-run');
+
+    var tasks = ['prompt:gh-release'];
+
+    if (dryRun) {
+      grunt.log.ok('Skipping build, test:unit and copy:dist tasks in dry-run');
```

Comment:
Yeah I thought about it, but I figured that the point of the dry-run for the `release` task would always be to see what tags and releases would get generated. Those tasks would just slow things down in that context.

I have no problem bringing them back in.